### DISCUSSION TITLE
17 maggio 2023

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/mdp2223#6] [Q&A] [answered]
In questa discussion a tema la traccia di esame del 17 maggio 2023 discuteremo di eventuali proposte di soluzione all'esame in questione

**Comment by Gio0203c** [comment, marked as answer]
[17-05-2023 - COMPLETO.pdf](https://github.com/sapienzastudentsnetwork/mdp2223/files/12988533/17-05-2023.-.COMPLETO.pdf)

**Comment by Frisbon** [comment]
Traccia senza soluzione
https://drive.google.com/file/u/0/d/1sOF3qsW5gDm5LZK8pv_loW2CW6iliBc3/view?usp=drive_web

**Comment by FeddyLix17** [comment]
[Walkthrough Esame 17 Maggio 2023](https://github.com/FeddyLix17/Computer-Science/tree/main/Primo%20Anno/Secondo%20Semestre/Metodologie%20di%20Programmazione/Esami%20precedenti/2023/Maggio/Esame%2017%20Maggio%202023)
